sql/moz-fx-data-shared-prod/marketing_suppression_list_external/campaign_monitor_suppression_list_v1/metadata.yaml

```
@@ -0,0 +1,16 @@
+friendly_name: Campaign Monitor Suppression List
+description: |-
+  This python script gets the suppression list from Campaign Monitor via their API.
+  The data then gets stored into BigQuery.
+owners:
+- leli@mozilla.com
+labels:
+  incremental: false
+  owner1: leli
+scheduling:
+  dag_name: bqetl_suppression_list
+  arguments:
+  - --api_key = {{ var.value.campaign_monitor_api_key }}
+  - --client_id = {{ var.value.campaign_monitor_client_id }}
```

Since these are bash arguments, you may need to remove the space around the `=`.  I don't think the pod operator will clean up the args
```suggestion
  arguments:
  - --api_key={{ var.value.campaign_monitor_api_key }}
  - --client_id={{ var.value.campaign_monitor_client_id }}
```

---

I also just remembered this guide which recommends the google secret manager for secrets https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27922811/WTMO+Developer+Guide#WTMODeveloperGuide-ManagingsecretsinGKEworkloads
What you're doing here should be fine though.  I'm just not sure if the other way is the recommended way for bigquery_etl since this is new to me.

---

Just for posterity, discussed the secrets management in slack and I think this would require non-trivial changes to the dag tooling which is out of scope for this change